tests/assert.js

```diff
@@ -0,0 +1,49 @@
+export function assert(actual, expected, message) {
+    if (arguments.length == 1)
```

Comment:
I know you only moved this around but...
```suggestion
    if (arguments.length === 1)
```